blivetgui/list_devices.py

```diff
@@ -71,6 +71,13 @@ def load_disks(self):
 
         disks = self.blivet_gui.client.remote_call("get_disks")
 
+        # hide protected and hidden disks in installer mode
+        # (this will hide USB drive with installation image, disks under FW RAID etc.)
+        if self.blivet_gui.installer_mode:
+            for disk in disks:
+                if disk.protected or disk.format.hidden:
+                    disks.remove(disk)
```

Comment:
No, I do want to display all devices in blivet-gui, even if format is hidden or protected -- at least to let the user know there is a device with protected format.
I am doing this only in installer mode so the behavior is the same in blivet-gui and custom spoke.